R/input-slider.R

```diff
@@ -79,8 +88,9 @@ sliderInput <- function(inputId, label, min, max, value, step = NULL,
                         round = FALSE, format = NULL, locale = NULL,
                         ticks = TRUE, animate = FALSE, width = NULL, sep = ",",
                         pre = NULL, post = NULL, timeFormat = NULL,
-                        timezone = NULL, dragRange = TRUE)
-{
+                        timezone = NULL, dragRange = TRUE,
+                        skin = c("shiny", "big", "flat", "modern", "round", "sharp", "square"),
```

Comment:
Perhaps the ordering here should reflect our reflect our preferences (i.e., flat and round are pretty nice)